misc/systemd/cf-runalerts.service.in

```diff
@@ -3,6 +3,7 @@ Description=CFEngine Enterprise SQL Alerts
 After=syslog.target
 ConditionPathExists=@bindir@/runalerts.php
 ConditionFileIsExecutable=@workdir@/httpd/php/bin/php
+ConditionPathIsDirectory=@workdir@/httpd/php/runalerts-stamp
```

Comment:
As with the masterfiles change, maybe Asserts instead so we get errors and not silent failures:

```suggestion
AssertPathExists=@bindir@/runalerts.php
AssertFileIsExecutable=@workdir@/httpd/php/bin/php
AssertPathIsDirectory=@workdir@/httpd/php/runalerts-stamp
```

Reply:
Doesn't it mean we will be getting errors before the directory shows up?

Reply:
Not sure. I'll create a ticket for us to work on it later.